### ISSUE TITLE
Update /enterprise visual design [July 8]

### ISSUE THREAD

**Issue by stephaniehobson** [mozilla/bedrock#7352]
## Description

Update the look of /enterprise to match the new brand.

Ideally in production July 8th for launch of Firefox 68 on July 9th.

![fx-enterprise-1312](https://user-images.githubusercontent.com/854701/60305353-177b2f00-98f1-11e9-9d1b-61862c37259f.png)

Details in:
https://trello.com/c/OdBsjECC/324-update-enterprise-visual-design-only-ui-dev-3

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] /enterprise landing page has updated look
- [ ] /enterprise/signup has updated look
- [ ] /enterprise/signup/thanks has updated look

#### :heavy_exclamation_mark: Risks :heavy_exclamation_mark: 
- [ ] @stephaniehobson is on PTO next week, July 8th is tight.
- [ ] signup and thanks depend on #7141 

#### Tasks
- [x] Update /enterprise page
- [x] Re-code /enterprise to remove the table
- [x] design review 
- [ ] code review 
- [ ] Update /signup and /thanks to use Protocol basic templates
- [ ] Tweak /signup and /thanks as needed
- [ ] code review

**Comment by stephaniehobson** [comment]
Submitted #7353 for the homepage. Signup and thanks pages will be in a separate PR.
